executor/src/test/java/org/k8loud/executor/action/kubernetes/HorizontalScalingActionTest.java

```diff
@@ -3,40 +3,32 @@
 import data.ExecutionExitCode;
 import data.ExecutionRS;
 import data.Params;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.api.Test;
 import org.k8loud.executor.action.Action;
 import org.k8loud.executor.exception.ActionException;
 import org.k8loud.executor.exception.KubernetesException;
 
 import java.util.Map;
-import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
 class HorizontalScalingActionTest extends BaseTest {
     private static final String REPLICAS_KEY = "replicas";
 
-    @ParameterizedTest
-    @MethodSource
-    void testIfKubernetesServiceIsCalled(Params params) throws ActionException, KubernetesException {
+    @Test
+    void testIfKubernetesServiceIsCalled() throws ActionException, KubernetesException {
+        // given
+        Params params = new Params(Map.of(NAMESPACE_KEY, NAMESPACE, RESOURCE_NAME_KEY, RESOURCE_NAME, RESOURCE_TYPE_KEY,
+                RESOURCE_TYPE, REPLICAS_KEY, "3"));
+
         // when
         Action action = new HorizontalScalingAction(params, kubernetesService);
         ExecutionRS rs = action.perform();
 
         // then
-        verify(kubernetesService, times(1)).scaleHorizontally(params.getRequiredParam(NAMESPACE_KEY),
-                params.getRequiredParam(RESOURCE_NAME_KEY), params.getRequiredParam(RESOURCE_TYPE_KEY),
-                params.getRequiredParamAsInteger(REPLICAS_KEY));
+        Integer replicas = params.getRequiredParamAsInteger(REPLICAS_KEY);
+        verify(kubernetesService).scaleHorizontally(NAMESPACE, RESOURCE_NAME, RESOURCE_TYPE, replicas);
```

Comment:
```suggestion
        verify(kubernetesService).scaleHorizontally(eq(NAMESPACE), eq(RESOURCE_NAME), eq(RESOURCE_TYPE), eq(3));
```

Reply:
SonarLint says it's a useless change, not adding

Reply:
then for me you should
```suggestion
        verify(kubernetesService).scaleHorizontally(NAMESPACE, RESOURCE_NAME, RESOURCE_TYPE, 3);
```